app/components/Header.js

```diff
@@ -64,9 +64,10 @@ const Header = () => {
                 </a>
                 <a
                   style={{marginLeft: '20px'}}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  href="http://localhost:3000/question/66"
+                  // (Dylan) Why these next 2 lines?
+                  // target="_blank"
+                  // rel="noopener noreferrer"
```

Comment:
@dleard yeah, you don't need both. You also only need to strip the opener property on a cross-origin request (like the original to Metabase) so if you're opening the same app it's fine to just `target="_blank"` 
https://developers.google.com/web/tools/lighthouse/audits/noopener